oauth2_http/java/com/google/auth/oauth2/ComputeEngineCredentials.java

```diff
@@ -114,6 +122,13 @@ public ComputeEngineCredentials(HttpTransportFactory transportFactory) {
   }
 
   /**
+   * Create a new ComputeEngineCredentials instance.
```

Comment:
Please copy here documentation from the deprecated `ComputeEngineCredentials()`

Reply:
I will make them consistent.